Dokumentationen/project_management/risk_list_neu.csv

```diff
@@ -0,0 +1,20 @@
+<project> Risk List;;;;;;;;;
+Risk ID;Date Identified;Headline;Description;Type;Impact;Probability;Magnitude;Owner;Mitigation Strategy
+1;01.12.2020;Datenschutz- Login Professoren;Die Daten der Dozenten m�ssen (Benutzername, Passwort) gesch�tzt werden. Die gebuchten Experimente sowie zus�tzliche Angaben sind nur f�r den Admin + dem jeweiligen Donzenten einsehbar.;Ressource?;5;20%;1,0;TW;Login Daten entsprechend sch�tzen.
+2;01.12.2020;Datenschutz- Lizenzen, Urheberrecht;Es besteht die Gefahr das Grafiken missbr�uchlich genutzt werden. Dies kann zu einer Urheberrechtsverletzung f�hren.;Ressource?;5;60%;3,0;TW;Lizenzvereinbarungen der neugenutzen Grafiken kl�ren oder eigene Grafiken erstellen
+3;21.12.2020;Missverst�ndinisse mit Auftraggeber;In den Gespr�chen mit dem Auftraggeber kommt/ kann es zu Missverst�ndnissen in der Kommunikation und im Verst�ndnis kommen. Diese werden vor allem durch einen unterschiedlichen Sprachgebrauch als auch durch unterschiedliche Vorstellungen der technichen Umsetzung und der technischen Sinnhaftigkeit  getragen.;Business;5;80%;4,0;TW, TK,GM, AB;H�ufige Meetings und reger Ausstausch mit dem Auftraggeber.Einfacher Sprachgebrauch und Vermeidung von Fachbegriffen. Optische Darstellungen nutzen um ein besseres Verst�ndnis zu erreichen.
+4;05.01.2021;Zeit f�r Bearbeitung reicht nicht aus;Es besteht das Risiko, dass der Umfang des Arbeitsaufwandes untersch�tzt wird und dadurch die rechtzeitige Fertigstellung in Gefahr ist.;Schedule;3;20%;0,6;GM;Deadlines f�r Meilensteine setzen. Im Team den Aufwand gemeinsam kritisch beurteilen.

```

Comment:
+ Iteratives Vorgehen - das wichtigste immer zuerst bearbeiten. Strenge Priorisierung = das Maximum wird in der gegebenen Zeit umgesetzt (Nutzenorientierung)